src/Bicep.MSBuild.E2eTests/examples/Directory.Build.props

```diff
@@ -0,0 +1,5 @@
+<!--
+  This project prevents the e2e tests from inheriting our repo build settings.
+-->
+<Project>
```

Comment:
>Project [](start = 1, length = 7)

The e2e tests for the msbuild task involve calling msbuild to build test projects. The files in this directory isolate the test builds from our normal settings.